api-spec/openapi.yaml

```
@@ -367,6 +365,27 @@ components:
           type: string
         statusDetails:
           type: string
+        eventStatus:
+          type: string
+          enum:
+            - ACTIVATED
+            - AUTHORIZATION_REQUESTED
+            - AUTHORIZATION_COMPLETED
+            - CLOSED
+            - CLOSURE_ERROR
+            - NOTIFIED_OK
+            - NOTIFIED_KO
+            - NOTIFICATION_ERROR
+            - NOTIFICATION_REQUESTED
+            - EXPIRED
+            - REFUNDED
+            - CANCELED
+            - EXPIRED_NOT_AUTHORIZED
+            - UNAUTHORIZED
+            - REFUND_ERROR
+            - REFUND_REQUESTED
+            - CANCELLATION_REQUESTED
+            - CANCELLATION_EXPIRED
```

What about using a ref to `transactions-service` `TransactionStatus`?
```suggestion
    $ref: "https://raw.githubusercontent.com/pagopa/pagopa-ecommerce-transactions-service/main/api-spec/transactions-api.yaml#/components/schemas/TransactionStatus"
```

---

I liked it